### PR TITLE
Phrases

### PR DESCRIPTION
**Pull request by mkusher**

### CODE REVIEW

**Comment by YuraLukashik** [comment]
fix tests!

**Comment by YuraLukashik** [comment]
@mkusher use labels to describe the state of PR!

**Comment by mkusher** [comment]
@YuraLukashik I have no permission to do it
It is ready to review & merge

File: src/Phrases.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default class Phrases {
+    constructor(phrases) {
```
Comment (YuraLukashik) [inline src/Phrases.js:2]: we need to add a restriction to the number of phrases!
like throwing error, when phrases.length <= 2

**Pull request merged**